week7/cooper_client_react/04_adding_a_user_class.md

```diff
@@ -56,7 +65,7 @@ end
 
 You can add more attributes to the User factory if you like, we added just the minimal required attributes at the moment.
 
-Let's add a spec for the User factory we just created.
+The Devise token auth generator does not create a unit spec for the models it generates. Let's add the user spec and test the User factory we just created. 
 ```
```

Comment:
move devise mount outside of namespaces

`mount_devise_token_auth_for 'User', at: 'api/v1/auth', skip: [:omniauth_callbacks]`

Reply:
you need to change `current_api_v1_user` to `current_user` everywhere it is used

Reply:
addressed